package.json

```diff
@@ -71,5 +71,8 @@
   "engines": {
     "node": ">= 0.10"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "validation-test-data": "^1.0.0"
```

Comment:
I would use the validatorjs organization npm scope:
`@validatorjs/test-data`

Reply:
Shouldn't the package whatever the name is added as a  dev dependency? End users don't need the new package in order to use `validator`